src/Compilers/Server/VBCSCompilerTests/NamedPipeClientConnectionHostTests.cs

```diff
@@ -57,13 +57,13 @@ public async Task CallAfterComplete()
         }
 
         [ConditionalFact(typeof(WindowsOrLinuxOnly), Reason = "https://github.com/dotnet/runtime/issues/40301")]
-        public void EndListenCancelsIncompleteTask()
+        public async Task EndListenCancelsIncompleteTask()
```

Comment:
Was this actually testing anything before?

Reply:
That is definitely questionable. I only noticed because one of the new xunit analyzers caught the bug.